src/tracing/errors.test.ts

```diff
@@ -11,17 +11,6 @@ describe('beforeSend', () => {
   const ERROR = {} as ErrorEvent
 
   describe('updateRequestUrl', () => {
-    test('should remove "/#" from the request URL', () => {
-      const event = {
-        request: {
-          url: 'https://app.uniswap.org/#/example',
-        },
-      } as ErrorEvent
-
-      beforeSend(event, {})
-      expect(event.request?.url).toBe('https://app.uniswap.org/example')
-    })
```

Comment:
are we removing this functionality? ie no longer handling converting from hash to path based urls? This will break a lot of downstream users who have hardcoded Uni urls

Reply:
good point - I added this back in.